python/cudf/bindings/unaryops.pyx

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+from .cudf_cpp cimport *
+from .cudf_cpp import *
+
+from librmm_cffi import librmm as rmm
+
+from libc.stdlib cimport free
+
+
+_MATH_OP = {}
+_MATH_OP['sin']     = GDF_SIN
+_MATH_OP['cos']     = GDF_COS
+_MATH_OP['tan']     = GDF_TAN
+_MATH_OP['asin']    = GDF_ARCSIN
+_MATH_OP['acos']    = GDF_ARCCOS
+_MATH_OP['atan']    = GDF_ARCTAN
+_MATH_OP['exp']     = GDF_EXP
+_MATH_OP['log']     = GDF_LOG
+_MATH_OP['sqrt']    = GDF_SQRT
+_MATH_OP['ceil']    = GDF_CEIL
+_MATH_OP['floor']   = GDF_FLOOR
+_MATH_OP['abs']     = GDF_ABS
+_MATH_OP['not']     = GDF_BIT_INVERT
```

Comment:
You should be able to define these at dictionary creation instead of afterwards.